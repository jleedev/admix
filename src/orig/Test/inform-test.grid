Marker Set: all markers
  M1    M2    M3   INFORM
0.000 0.000 1.000 ********
0.000 0.050 0.950 ********
0.000 0.100 0.900 ********
0.000 0.150 0.850 ********
0.000 0.200 0.800 ********
0.000 0.250 0.750 ********
0.000 0.300 0.700 ********
0.000 0.350 0.650 ********
0.000 0.400 0.600 ********
0.000 0.450 0.550 ********
0.000 0.500 0.500 ********
0.000 0.550 0.450 ********
0.000 0.600 0.400 ********
0.000 0.650 0.350 ********
0.000 0.700 0.300 ********
0.000 0.750 0.250 ********
0.000 0.800 0.200 ********
0.000 0.850 0.150 ********
0.000 0.900 0.100 ********
0.000 0.950 0.050 ********
0.000 1.000 0.000 ********
0.050 0.000 0.950 ********
0.050 0.050 0.900 1381.57
0.050 0.100 0.850 837.026
0.050 0.150 0.800 625.957
0.050 0.200 0.750 512.47
0.050 0.250 0.700 442.275
0.050 0.300 0.650 395.588
0.050 0.350 0.600 363.433
0.050 0.400 0.550 341.204
0.050 0.450 0.500 326.393
0.050 0.500 0.450 317.66
0.050 0.550 0.400 314.442
0.050 0.600 0.350 316.834
0.050 0.650 0.300 325.691
0.050 0.700 0.250 343.045
0.050 0.750 0.200 373.252
0.050 0.800 0.150 426.349
0.050 0.850 0.100 530.566
0.050 0.900 0.050 807.71
0.050 0.950 0.000 ********
0.100 0.000 0.900 ********
0.100 0.050 0.850 969.029
0.100 0.100 0.800 611.575
0.100 0.150 0.750 470.171
0.100 0.200 0.700 392.948
0.100 0.250 0.650 344.8
0.100 0.300 0.600 312.813
0.100 0.350 0.550 291.089
0.100 0.400 0.500 276.62
0.100 0.450 0.450 267.825
0.100 0.500 0.400 263.971
0.100 0.550 0.350 264.966
0.100 0.600 0.300 271.376
0.100 0.650 0.250 284.729
0.100 0.700 0.200 308.361
0.100 0.750 0.150 349.934
0.100 0.800 0.100 430.598
0.100 0.850 0.050 637.686
0.100 0.900 0.000 ********
0.150 0.000 0.850 ********
0.150 0.050 0.800 783.274
0.150 0.100 0.750 507.102
0.150 0.150 0.700 397.469
0.150 0.200 0.650 337.355
0.150 0.250 0.600 299.925
0.150 0.300 0.550 275.325
0.150 0.350 0.500 259.084
0.150 0.400 0.450 248.97
0.150 0.450 0.400 243.902
0.150 0.500 0.350 243.551
0.150 0.550 0.300 248.26
0.150 0.600 0.250 259.249
0.150 0.650 0.200 279.32
0.150 0.700 0.150 314.954
0.150 0.750 0.100 383.925
0.150 0.800 0.050 558.155
0.150 0.850 0.000 ********
0.200 0.000 0.800 ********
0.200 0.050 0.750 674.196
0.200 0.100 0.700 444.456
0.200 0.150 0.650 353.669
0.200 0.200 0.600 304.047
0.200 0.250 0.550 273.417
0.200 0.300 0.500 253.71
0.200 0.350 0.450 241.318
0.200 0.400 0.400 234.526
0.200 0.450 0.350 232.654
0.200 0.500 0.300 235.79
0.200 0.550 0.250 244.902
0.200 0.600 0.200 262.416
0.200 0.650 0.150 294.058
0.200 0.700 0.100 355.545
0.200 0.750 0.050 510.027
0.200 0.800 0.000 ********
0.250 0.000 0.750 ********
0.250 0.050 0.700 602.784
0.250 0.100 0.650 402.947
0.250 0.150 0.600 324.72
0.250 0.200 0.550 282.345
0.250 0.250 0.500 256.621
0.250 0.300 0.450 240.65
0.250 0.350 0.400 231.442
0.250 0.400 0.350 227.708
0.250 0.450 0.300 229.192
0.250 0.500 0.250 236.591
0.250 0.550 0.200 252.018
0.250 0.600 0.150 280.657
0.250 0.650 0.100 336.848
0.250 0.700 0.050 478.301
0.250 0.750 0.000 ********
0.300 0.000 0.700 ********
0.300 0.050 0.650 553.512
0.300 0.100 0.600 374.254
0.300 0.150 0.550 305.001
0.300 0.200 0.500 268.033
0.300 0.250 0.450 246.187
0.300 0.300 0.400 233.413
0.300 0.350 0.350 227.225
0.300 0.400 0.300 226.786
0.300 0.450 0.250 232.439
0.300 0.500 0.200 245.991
0.300 0.550 0.150 272.194
0.300 0.600 0.100 324.422
0.300 0.650 0.050 456.928
0.300 0.700 0.000 ********
0.350 0.000 0.650 ********
0.350 0.050 0.600 518.848
0.350 0.100 0.550 354.319
0.350 0.150 0.500 291.799
0.350 0.200 0.450 259.138
0.350 0.250 0.400 240.637
0.350 0.300 0.350 230.926
0.350 0.350 0.300 228.023
0.350 0.400 0.250 231.704
0.350 0.450 0.200 243.405
0.350 0.500 0.150 267.49
0.350 0.550 0.100 316.628
0.350 0.600 0.050 442.937
0.350 0.650 0.000 ********
0.400 0.000 0.600 ********
0.400 0.050 0.550 494.732
0.400 0.100 0.500 340.973
0.400 0.150 0.450 283.723
0.400 0.200 0.400 254.734
0.400 0.250 0.350 239.418
0.400 0.300 0.300 233.03
0.400 0.350 0.250 234.241
0.400 0.400 0.200 243.917
0.400 0.450 0.150 266.006
0.400 0.500 0.100 312.638
0.400 0.550 0.050 434.745
0.400 0.600 0.000 ********
0.450 0.000 0.550 ********
0.450 0.050 0.500 478.891
0.450 0.100 0.450 333.062
0.450 0.150 0.400 280.131
0.450 0.200 0.350 254.549
0.450 0.250 0.300 242.643
0.450 0.300 0.250 240.399
0.450 0.350 0.200 247.598
0.450 0.400 0.150 267.61
0.450 0.450 0.100 312.096
0.450 0.500 0.050 431.529
0.450 0.550 0.000 ********
0.500 0.000 0.500 ********
0.500 0.050 0.450 470.113
0.500 0.100 0.400 330.086
0.500 0.150 0.350 280.925
0.500 0.200 0.300 258.878
0.500 0.250 0.250 251.171
0.500 0.300 0.200 254.951
0.500 0.350 0.150 272.519
0.500 0.400 0.100 314.997
0.500 0.450 0.050 432.979
0.500 0.500 0.000 ********
0.550 0.000 0.450 ********
0.550 0.050 0.400 467.945
0.550 0.100 0.350 332.089
0.550 0.150 0.300 286.558
0.550 0.200 0.250 268.753
0.550 0.250 0.200 267.123
0.550 0.300 0.150 281.377
0.550 0.350 0.100 321.697
0.550 0.400 0.050 439.214
0.550 0.450 0.000 ********
0.600 0.000 0.400 ********
0.600 0.050 0.350 472.624
0.600 0.100 0.300 339.719
0.600 0.150 0.250 298.267
0.600 0.200 0.200 286.535
0.600 0.250 0.150 295.529
0.600 0.300 0.100 333.034
0.600 0.350 0.050 450.833
0.600 0.400 0.000 ********
0.650 0.000 0.350 ********
0.650 0.050 0.300 485.222
0.650 0.100 0.250 354.541
0.650 0.150 0.200 318.763
0.650 0.200 0.150 317.733
0.650 0.250 0.100 350.671
0.650 0.300 0.050 469.122
0.650 0.350 0.000 ********
0.700 0.000 0.300 ********
0.700 0.050 0.250 508.127
0.700 0.100 0.200 379.887
0.700 0.150 0.150 354.3
0.700 0.200 0.100 377.98
0.700 0.250 0.050 496.567
0.700 0.300 0.000 ********
0.750 0.000 0.250 ********
0.750 0.050 0.200 546.329
0.750 0.100 0.150 423.392
0.750 0.150 0.100 422.644
0.750 0.200 0.050 538.161
0.750 0.250 0.000 ********
0.800 0.000 0.200 ********
0.800 0.050 0.150 611.22
0.800 0.100 0.100 506.771
0.800 0.150 0.050 605.271
0.800 0.200 0.000 ********
0.850 0.000 0.150 ********
0.850 0.050 0.100 735.263
0.850 0.100 0.050 730.687
0.850 0.150 0.000 ********
0.900 0.000 0.100 ********
0.900 0.050 0.050 1070.54
0.900 0.100 0.000 ********
0.950 0.000 0.050 ********
0.950 0.050 0.000 ********
1.000 0.000 0.000 ********
