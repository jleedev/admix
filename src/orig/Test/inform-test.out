Marker Set: all markers
  M1    M2    M3   INFORM
0.150 0.600 0.250 259.249
0.470 0.210 0.320 252.713
0.800 0.005 0.195 3232.03

Marker Set: CSF1PO D13S317
  M1    M2    M3   INFORM
0.150 0.600 0.250 6.92354
0.470 0.210 0.320 6.29442
0.800 0.005 0.195 85.1943

Marker Set: D16S539 D18S51 D21S11 D3S1358 D5S818 D7S820 D8S1179 FGA
  M1    M2    M3   INFORM
0.150 0.600 0.250 96.6003
0.470 0.210 0.320 81.9826
0.800 0.005 0.195 1040.56

Marker Set: TH01
  M1    M2    M3   INFORM
0.150 0.600 0.250 1.60013
0.470 0.210 0.320 1.75614
0.800 0.005 0.195 2.13298

Marker Set: TPOX vWA
  M1    M2    M3   INFORM
0.150 0.600 0.250 4.05575
0.470 0.210 0.320 5.69113
0.800 0.005 0.195 84.0331
